src/class-parsely.php

```diff
@@ -2057,4 +2057,45 @@ public function parsely_is_user_logged_in() {
 	public function convert_jsonld_to_parsely_type( $type ) {
 		return in_array( $type, $this->supported_jsonld_post_types ) ? 'post' : 'index';
 	}
+
+	/**
+	 * Determine if an API key is saved in the options.
+	 *
+	 * @since 2.6.0
+	 *
+	 * @return bool True is API key is set, false if it is missing.
+	 */
+	public function api_key_is_set() {
+		$options = $this->get_options();
+
+		return (
+				isset( $options['apikey'] ) &&
+				is_string( $options['apikey'] ) &&
+				'' !== $options['apikey']
```

Comment:
Is there any reason not to use `empty()` here?

Reply:
`empty()` is a function call, which is more expensive than a comparison as I understand it.

Comparing against `''` is also more descriptive - since `empty()` checks against multiple things - for instance, if `$options['apikey` was a literal string of `0`, then it's set, it's a string, and `empty( '0' )` would be true, but `'' !== '0'` would be correctly false.